app/src/main/java/timur/gilfanov/messenger/domain/usecase/user/repository/ApplyRemoteSettingsRepositoryError.kt

```diff
@@ -0,0 +1,5 @@
+package timur.gilfanov.messenger.domain.usecase.user.repository
+
+sealed interface ApplyRemoteSettingsRepositoryError {
+    data object SettingsNotApplied : ApplyRemoteSettingsRepositoryError
```

Comment:
Must be more specific, so UI can handle error properly.
```suggestion
    data class Failed(val transient: Boolean) : ApplyRemoteSettingsRepositoryError
    data class Cooldown(val remains: Duration) : ApplyRemoteSettingsRepositoryError
    data object Unauthorized(): ApplyRemoteSettingsRepositoryError
```